.eslintignore

```diff
@@ -0,0 +1,4 @@
+out
```

Comment:
Acho que não precisa desse arquivo já que o comando de lint manda rodar só dentro da pasta src

Reply:
É que existem arquivos js/ts fora da pasta src que também dão warn se quebrarem, tipo os de configuração como o next.config.js. Nesse caso eu utilizei o eslint . --fix, e aí precisou do eslintignore, mas posso remover.

Reply:
Sem problemas, se for assim então melhor atualizar o comando no `package.json`